src/sidebar/components/Annotation/AnnotationHeader.tsx

```diff
@@ -54,6 +55,7 @@ function AnnotationHeader({
   const defaultAuthority = store.defaultAuthority();
   const displayNamesEnabled = store.isFeatureEnabled('client_display_names');
   const userURL = store.getLink('user', { user: annotation.user });
+  const pageNumbersEnabled = store.isFeatureEnabled('page_numbers');
```

Comment:
The `PageSelector` selector uses the term "label" rather than "number", to create a clear distinction from the index. However I use the term "page number" in code and the feature name because that is a phrase that is more obvious for users.